diff --git a/03.broken_function.py b/.answers/03.broken_function.py
index f2a0117..a7f45a2 100644
--- a/03.broken_function.py
+++ b/.answers/03.broken_function.py
@@ -27,7 +27,7 @@ It's taken pretty directly from the MyPy docs
 """
 
 
-def hello(name):
+def hello(name: str) -> str:
     return f'hello {name}'
 
 
@@ -42,7 +42,10 @@ for case in TEST_CASES:
     print(hello(case))
 
 
-def extension_excercise(name):
+from typing import Union, List
+
+
+def extension_excercise(name: Union[str, list ]) -> str:
     """If you fancy a challenge try doing the same here...
 
     (
